pkg/kube/controller/controller.go

```diff
@@ -0,0 +1,151 @@
+// Copyright Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package controller
```

Comment:
Wouldn't it be more unobtrusive to have this moved under `pkg/servicemesh/kube/controller/` instead if there's nothing in upstream we could use yet?

Reply:
I like this idea.

Reply:
+1 to that

Reply:
Longer stretch - moving stuff from `servicemesh` folder to sth else. After all, istio IS service mesh, so that name is not unique enough ;)

Done in 03dc2e251e